DependencyInjection/Configuration.php

```diff
@@ -9,24 +9,26 @@ class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('cross_knowledge_device_detect');
+        $treeBuilder = new TreeBuilder('cross_knowledge_device_detect');
+        $rootNode = $treeBuilder->getRootNode();
 
         $rootNode
             ->children()
-                ->scalarNode('cache_manager')->info('The service name that will handle caching (must implement Doctrine\Common\Cache\CacheProvider))')
-                ->end()
-                ->arrayNode('device_detector_options')
-                    ->info("Available options are discard_bot_information and skip_bot_detection which are booleans")
-                        ->children()
-                            ->booleanNode('discard_bot_information')
-                                ->defaultTrue()
-                            ->end()
-                            ->booleanNode('skip_bot_detection')
-                                ->defaultTrue()
-                            ->end()
-                        ->end()
-                ->end()
+            ->scalarNode('cache_manager')->info(
```

Comment:
for me it was more clear with the old formatting

Reply:
Ok. I will change it.